packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -492,6 +492,35 @@ export type InsertableField<TSchema extends ImplicitFieldSchema | UnsafeUnknownS
 		? InsertableContent | undefined
 		: never;
 
+/**
+ * Content which could be read from a field within a tree.
+ *
+ * @remarks
+ * Extended version of {@link TreeFieldFromImplicitField} that also allows {@link (UnsafeUnknownSchema:type)}.
+ * Since reading from fields with non-exact schema is still safe, this is only useful (compared to {@link TreeFieldFromImplicitField}) when the schema is also used as input and thus allows {@link (UnsafeUnknownSchema:type)}
+ * for use
+ * @system @alpha
+ */
+export type ReadableField<TSchema extends ImplicitFieldSchema | UnsafeUnknownSchema> =
+	TSchema extends ImplicitFieldSchema
+		? TreeFieldFromImplicitField<TSchema>
+		: TreeLeafValue | TreeNode;
+
+/**
+ * Content which could be read from a field within a tree.
```

Comment:
Update doc - this was copied from the other one

Reply:
```suggestion
 * Adapter to remove {@link (UnsafeUnknownSchema:type)} from a schema type so it can be used with types for generating APIs for reading data.
```